.github/workflows/backup.yaml

```diff
@@ -0,0 +1,29 @@
+name: Backup Routine
+on:
+  workflow_dispatch:
+  schedule:
+    # Runs at 04:00 UTC on the 1st and 17th of every month
+    - cron: "0 4 */16 * *"
+jobs:
+  backup-production-dataset:
+    runs-on: ubuntu-20.04
+    name: Backup production dataset
+    steps:
```

Comment:
rart den ikke trengte noen permissions, men ser den er lik og kjørt ok i familie-sanity-brev

Reply:
Alle som vi kan lese data fra datasettet: https://avzz8jwq.api.sanity.io/v2023-03-01/data/query/production?query=*%0A&perspective=published. Det er derfor man ikke trenger to token. Det er kun om man ønsker å skrive man må ha det.

Reply:
Ser vi setter token på linje 19 da 🤔
`SANITY_AUTH_TOKEN="${{ secrets.SANITY_VIEWER_TOKEN }}" `

Reply:
Det tokenet er vel for å lese fra sanity? Og er lagt til repoet her